src/releng_channel_status/releng_channel_status/templates/channel_status.html

```diff
@@ -0,0 +1,34 @@
+{% extends "layout.html" %}
+{% block body %}
+    <div class="row">
+        <div class="col">
+            <h1 class="display-4">{{ channel_status.product }}/{{ channel_status.channel }}</h1>
+        </div>
+    </div>
+
+    {% if channel_status.is_latest_build_update %}
+        <div class="alert alert-success" role="alert">
+            <span class="text-capitalize"> {{ channel_status.channel }}</span> updates are currently.
```

Comment:
Nit 1: I realize a better sentence can be `{{ channel_status.channel }} channel is currently`. Sorry for bad wording of mine in bug 1386307

Nit 2: I think `{{ channel_status.channel }} channel is currently` should be displayed outside of the alert div. The important part of the sentence is `serving the latest nightly ([Build ID])`. So in my opinion, that's what should be in the alert div.

Nit 3: My apologies, I left a confusion about what's in the `if` and in the `else`. I believe the logic should be:

```html
<span class="text-capitalize"> {{ channel_status.channel }}</span> updates are currently

{% if channel_status.is_latest_build_update %}
<div class="alert alert-success" role="alert">
  <h2 class="alert-heading">serving the latest {{ channel_status.channel }} {{channel_status.rule['buildID']}}</h2>
</div>
{% else %}
<div class="alert alert-warning" role="alert">
  <h2 class="alert-heading">frozen to {{channel_status.rule['buildID']}}</h2>
  <p>Reason: {{ channel_status.rule['comment'] }}</p>
</div>
{% endif %}
```

Reply:
I'm not sure if the buildID of the rule is the right place to get the latest update buildID, the current nightly has no buildID:

`{
alias: "firefox-nightly",
backgroundRate: 100,
buildID: null,
buildTarget: null,
channel: "nightly",
comment: null,
data_version: 74,
distVersion: null,
distribution: null,
fallbackMapping: null,
headerArchitecture: null,
instructionSet: null,
locale: null,
mapping: "Firefox-mozilla-central-nightly-latest",
memory: null,
mig64: null,
osVersion: null,
priority: 90,
product: "Firefox",
rule_id: 3,
update_type: "minor",
version: null
}`

Reply:
@JohanLorenzo 
Should we consider the throttle in this page, or just look to mapping and not to fallbackMapping?

Reply:
> Should we consider the throttle in this page, or just look to mapping and not to fallbackMapping?
Yes, the throttle is important. As you may know, a rule with:
```json
backgroundRate: 0,
mapping: "Firefox-mozilla-central-nightly-latest",
fallbackMapping: "Firefox-mozilla-central-nightly-20171015220106",
```
won't automatically serve the latest nightly. Then, we can't only rely on the mapping.

> I'm not sure if the buildID of the rule is the right place to get the latest update buildID,
You're right, getting the buildID of "Firefox-mozilla-central-nightly-latest" is tricky, actually. I initially thought of fetching https://aus-api.mozilla.org/api/v1/releases/Firefox-mozilla-central-nightly-latest and looking into all locales in all platforms to see if one buildId stood out. But that's not a good idea, after all. This blob keeps changing during the day, depending on how fast files are updated.
I also thought of comparing `Firefox-mozilla-central-nightly-latest` to the latest known release (for instance Firefox-mozilla-central-nightly-20171015220106), but if some locales are missing, the JSON will be differents.

@mozbhearsum, would you have an idea on how to fetch the latest buildId of a release?